pertpy/tools/_differential_gene_expression/_simple_tests.py

```diff
@@ -94,9 +96,28 @@ def compare_groups(
         paired_by: str | None = None,
         mask: str | None = None,
         layer: str | None = None,
+        n_permutations: int = 1000,
+        permutation_test_statistic: type["SimpleComparisonBase"] | None = None,
         fit_kwargs: Mapping = MappingProxyType({}),
         test_kwargs: Mapping = MappingProxyType({}),
     ) -> DataFrame:
+        """Perform a comparison between groups.
+
+        Args:
```

Comment:
Please remove all typing from the docstring here. We only type the function signature which is enough for Sphinx.